manifests/config.pp

```diff
@@ -86,4 +80,34 @@
       match => '^SELINUXTYPE=\w+',
     }
   }
+
+  # Module build config:
+  validate_absolute_path($module_build_root)
+
+  file {$module_build_root:
+    ensure => 'directory',
+    owner  => 'root',
+    group  => 'root',
+    mode   => '0644',
+  }
+
+  # put helper in place:
+  file {"${module_build_root}/modules/selinux_build_module.sh":
```

Comment:
I think this file needs `bin_t`seltype or puppet agent will maybe throw SELinux AVC's when it executes?